### PR TITLE
GSoC2024 `Added my solution for the DL starter problem`

### PR DESCRIPTION
**Pull request by Shashankss1205**
**Added my solutions for the gsoc'24 toy test for ML/DL by Nasa, solving Issue #243**

Description and Analysis:
This PR demonstrates how I systematically explored galaxy measurement interpolation methods, including **linear** and **polynomial interpolation**, **PyTorch-based neural networks (ML/DL)**, and **support vector regression model**. Through meticulous evaluation and visual confirmations, I showcased their efficacy and accuracy in predicting measurements at desired wavelengths. 

`Linear Interpolation Results`

Screenshot from 2024-03-14 12-19-41

`Polynomial Interpolation Results`

Screenshot from 2024-03-14 12-19-49

`DL-ML based Interpolation Results`

Screenshot from 2024-03-14 12-19-55

`Support Vector Machine based Interpolation Results`

Screenshot from 2024-03-14 12-20-01

### CODE REVIEW

**Comment by jkrick** [comment]
This was a starter problem for GSoC2024 and not meant to be merged.
